server/controllers/jobController.js

```diff
@@ -124,10 +126,133 @@ const deleteJobById = async (req, res) => {
   }
 };
 
+const addQuestions = async (req, res) => {
```

Comment:
Change the naming to addCustomQuestions and similarly for other routes, as it will be confused with questionController.js

Reply:
Yaa I will change it

Reply:
Do resolve after you make the change